README.md

```diff
@@ -1,12 +1,10 @@
 # Accessible autocomplete
 
----
-
-**We cannot currently respond to issues or pull requests.**
+The accessible autocomplete is a component that helps users choose countries and territories in answers.
 
-You can still raise issues or pull requests if you want to. As soon as we’re able to, we will prioritise dealing with any bugs that have been raised by people in the UK public sector. We cannot prioritise adding new features.
+Do not use the accessible autocomplete for services where users have to provide their country or territory.
```

Comment:
Just checking, what would we say if a service team needed to use a custom country/territory list?

Would we tell them adapt the 'country and territory autocomplete' or is 'accessible autocomplete' somehow better for customising?

Reply:
@calvin-lau-sig7 when you use the `country and territory autocomplete` you can provide your own list of countries.

Alternatively (what my team has done) a team could use the default list, but filter out ones they don't want.

Reply:
Thanks @edwardhorsford, that makes things clear, cheers!